lib/ansible/modules/system/known_hosts.py

```diff
@@ -175,34 +174,32 @@ def enforce_state(module, params):
 
     return params
 
-def sanity_check(module,host,key,sshkeygen):
+
+def sanity_check(module, host, key, sshkeygen):
     '''Check supplied key is sensible
 
     host and key are parameters provided by the user; If the host
     provided is inconsistent with the key supplied, then this function
     quits, providing an error to the user.
     sshkeygen is the path to ssh-keygen, found earlier with get_bin_path
     '''
-    #If no key supplied, we're doing a removal, and have nothing to check here.
+    # If no key supplied, we're doing a removal, and have nothing to check here.
     if key is None:
         return
-    #Rather than parsing the key ourselves, get ssh-keygen to do it
-    #(this is essential for hashed keys, but otherwise useful, as the
-    #key question is whether ssh-keygen thinks the key matches the host).
+    # Rather than parsing the key ourselves, get ssh-keygen to do it
+    # (this is essential for hashed keys, but otherwise useful, as the
+    # key question is whether ssh-keygen thinks the key matches the host).
 
-    #The approach is to write the key to a temporary file,
-    #and then attempt to look up the specified host in that file.
+    # The approach is to write the key to a temporary file,
+    # and then attempt to look up the specified host in that file.
     try:
         outf = tempfile.NamedTemporaryFile(mode='w+')
         outf.write(key)
         outf.flush()
     except IOError:
         e = get_exception()
-        module.fail_json(msg="Failed to write to temporary file %s: %s" % \
-                             (outf.name,str(e)))
-    rc,stdout,stderr=module.run_command([sshkeygen,'-F',host,
-                                         '-f',outf.name],
-                                        check_rc=True)
+        module.fail_json(msg="Failed to write to temporary file %s: %s" % (outf.name, str(e)))
+    rc, stdout, stderr = module.run_command([sshkeygen, '-F', host, '-f', outf.name], check_rc=True)
```

Comment:
This is a very long line (and wraps in my editor). Are you sure this is an improvement?